README.md

```diff
@@ -1,20 +1,56 @@
 <p align="center">
-	<img width=256px src="GitHub/logo.png" />
+    <img width=256px src="GitHub/logo.png" />
 </p>
 
 # Dynamic FPS
 
-Dynamic FPS automatically reduces the speed at which Minecraft renders when it's not focused (to 1 FPS) or hidden (no renders at all). It also fixes a bug in Vanilla Minecraft that makes it take much more performance in the background than it should.
+Dynamic FPS can reduce the frame rate, volume, and pause toasts while Minecraft is not focused or hidden.  
+In addition to this it also fixes a bug in vanilla Minecraft making it take too many resources in the background.
 
-Download in [the releases section](https://github.com/juliand665/Dynamic-FPS/releases), on [Modrinth](https://modrinth.com/mod/dynamic-fps), or on [CurseForge](https://www.curseforge.com/minecraft/mc-mods/dynamic-fps).
+Download on [GitHub Releases](https://github.com/juliand665/Dynamic-FPS/releases), [Modrinth](https://modrinth.com/mod/dynamic-fps), or [CurseForge](https://www.curseforge.com/minecraft/mc-mods/dynamic-fps).
+
+## Features
+
+Dynamic FPS will detect when the window is not focused, but instead either hovered over, unfocused, or invisible.  
+For each mode you can adjust the maximum FPS, master volume, toast visibility, and whether to invoke garbage collection.
+
+In addition to this Dynamic FPS also reduces background CPU usage and stops rendering the world while either the loading overlay is active or a screen is obscuring the world (you're in eg. the keybinds or any detected screen).
```

Comment:
i'd switch this around and correctly spell `e.g.`: "(e.g. when you're editing keybinds or in another screen that the mod detects as opaque)"